src/webgpu/api/validation/copyTextureToTexture.spec.ts

```diff
@@ -244,14 +259,34 @@ g.test('multisampled_copy_restrictions')
     );
   });
 
-g.test('uncompressed_texture_format_equality')
+g.test('texture_format_equality')
   .params(
     params()
-      .combine(poptions('srcFormat', kUncompressedTextureFormats))
-      .combine(poptions('dstFormat', kUncompressedTextureFormats))
+      .combine(poptions('srcFormat', kAllTextureFormats))
+      .combine(poptions('dstFormat', kAllTextureFormats))
   )
   .fn(async t => {
     const { srcFormat, dstFormat } = t.params;
+    const extensions: Array<GPUExtensionName> = [];
+
+    const srcFormatExtension = kAllTextureFormatInfo[srcFormat].extension;
+    if (srcFormatExtension !== undefined) {
+      if (!(await t.IsExtensionSupported(srcFormatExtension))) {
```

Comment:
It is unfortunate that an await is required here, maybe we could just set it on the descriptor, and see if asyncReinitDeviceWithDescriptor succeeds.

Reply:
Hi @kainino0x , sorry that I am not sure if this usage is correct or not. Could you take a look?